ansible/decommission/archive-logs.yml

```diff
@@ -74,11 +72,12 @@
         from {{ ansible_hostname }}:{{ item.expanded_paths | join(',') }}
     with_items: "{{_decommission_archive_management.results}}"
 
-
 - hosts: "{{ idr_environment | default('idr') }}-searchengine-hosts"
   vars:
     logs:
       - { name: 'searchengine', path: '/data/searchengine/searchengine/logs' }
+      # We may move archive the searchengine indexed data snapshot to a separate playbook
```

Comment:
I don't understand this comment.
Is this a something that should be captured as an issue for the future? Under what conditions should we or should we not archive the backup on decommissioning?